docs/download.html

```diff
@@ -0,0 +1,546 @@
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+        <li class="ov-nav-main__sub-item">Older Release Notes</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Data Warehouse Guide</a></li>
+        <li class="ov-nav-main__sub-item"><a class="ov-nav-main__sub-item__link" href="#">oVirt Metrics Store Installation Guide</a></li>
+      </ul>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="developers.html">Developers</a>
+    </li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="community.html">Community</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="https://lists.ovirt.org/archives/">Forum</a></li>
+    <li class="ov-nav-main__item"><a class="ov-nav-main__link" href="blog.html">Blog</a></li>
+  </ul>
+</nav>
+  </header>
+  <main class="ov-main ov-main--home">
+    <section class="ov-masthead ov-masthead--download">
+      <h1 class="ov-heading ov-heading--xl">Download oVirt</h1>
+      <h2 class="ov-heading ov-heading--lg">The preferred way to install oVirt is by downloading the Self-Hosted Engine.</h2>
+      <a href="" class="ov-button ov-button--primary">Install Self-Hosted Engine</a>
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">oVirt Quickstart Guide</h2>
+        <p>
+          oVirt 4.2.7 is intended for production use and is available for the following platforms:
+        </p>
+        <ul>
+          <li>
+            Red Hat Enterprise Linux 7.5 or later
+          </li>
+          <li>
+            CentOS Linux 7.5 or later
+          </li>
+          <li>
+            Scientific Linux 7.5 or later
+          </li>
+        </ul> 
+        <p>
+          See the Release Notes for oVirt 4.2.7 and the Installation Guide.
+        
+          oVirt can be downloaded and installed two primary ways: Self-Hosted Engine or separate Engine and Hosts.
+        
+          For production installations, we recommend installing oVirt using the Self-Hosted Engine configuration. In this configuration, oVirt Engine and a Host are installed together with the Engine running as a Virtual Machine on that Host. This configuration is more resilent because the Engine Virtual Machine will be highly available (once a second Host is added). See the Self-Hosted Engine Guide for full installation instructions for this configuration.
+        
+          If you prefer to run oVirt Engine standalone on physical hardware, you can install oVirt Engine and Nodes / Hosts separately. See the oVirt Installation Guide for full installation instructions for this configuration.
+        </p> 
+    </section>
+
+    <section class="ov-layout-styles ov-content-styles">
+      <h2 class="ov-heading ov-heading--lg">Download and Install Self-Hosted Engine</h2>
+      <div>
```

Comment:
You can probably drop the `div`

Reply:
Can do, I'll get rid of it across the board.